librz/core/cmd_descs/cmd_cmp.yaml

```diff
@@ -157,6 +157,11 @@ commands:
     args:
       - name: hexpair
         type: RZ_CMD_ARG_TYPE_STRING
+    details:
+      - name: Ignoring bytes
+        entries:
+          - text: "Ignore specific bytes while comparing by using '.'"
+            comment: "cx 1ee7..0f....d34d"
```

Comment:
I suggest to use the "regular" `Examples` section.
Also, the command `cx` should co in `text`, the argument `1ee7..0f....d34d` in `arg_str` and `"Ignore specific bytes while comparing by using '.'"` in `comment`.